README.md

```diff
@@ -41,6 +42,7 @@ const wallet = new Wallet({ chain: Chain.near, networkName: Network.testnet })
 wallet: new Wallet({
```

Comment:
- wallet declaration is duplicate. Remove previous one.
- Fix the declaration with: 
```js
const wallet = new Wallet({
  networkName: Wallet.testnet,
  chain: Wallet.near,
  // Get this in the developer tab under your profile image
  apiKey: YOUR_MINTBASEJS_API_KEY,
}),
```